packages/bundler-plugin-core/src/debug-id-upload.ts

```diff
@@ -47,34 +46,41 @@ export function debugIdUploadPlugin({
   sentryCliOptions,
   rewriteSourcesHook,
   deleteFilesAfterUpload,
-}: DebugIdUploadPluginOptions): UnpluginOptions {
-  return {
-    name: "sentry-debug-id-upload-plugin",
-    async writeBundle() {
-      let folderToCleanUp: string | undefined;
+}: DebugIdUploadPluginOptions) {
+  return async (buildArtifactPaths: string[]) => {
+    let folderToCleanUp: string | undefined;
 
-      const cliInstance = new SentryCli(null, sentryCliOptions);
+    const cliInstance = new SentryCli(null, sentryCliOptions);
 
-      try {
-        const tmpUploadFolder = await fs.promises.mkdtemp(
-          path.join(os.tmpdir(), "sentry-bundler-plugin-upload-")
-        );
+    try {
+      const tmpUploadFolder = await fs.promises.mkdtemp(
+        path.join(os.tmpdir(), "sentry-bundler-plugin-upload-")
+      );
 
-        folderToCleanUp = tmpUploadFolder;
+      folderToCleanUp = tmpUploadFolder;
 
-        const debugIdChunkFilePaths = (
-          await glob(assets, {
-            absolute: true,
-            nodir: true,
-            ignore: ignore,
-          })
-        ).filter(
-          (debugIdChunkFilePath) =>
-            debugIdChunkFilePath.endsWith(".js") ||
-            debugIdChunkFilePath.endsWith(".mjs") ||
-            debugIdChunkFilePath.endsWith(".cjs")
-        );
+      const debugIdChunkFilePaths = (
+        await glob(assets ?? buildArtifactPaths, {
```

Comment:
Can we add a log that we're using the auto-determined paths if we fall back to `buildArtifactPaths`?

Reply:
debug or info level?